src/xcode/ENA/ENA/Source/RootCoordinator.swift

```diff
@@ -20,6 +20,7 @@ protocol CoordinatorDelegate: AnyObject {
 	Should be used as a delegate in view controllers that need to communicate with other view controllers, either for navigation, or something else (e.g. transferring state).
 	Helps to decouple different view controllers from each other and to remove navigation responsibility from view controllers.
 */
+// swiftlint:disable:next type_body_length
```

Comment:
Is there no other way?

Reply:
Was shortly thinking about splitting. But its the RootCoordinator 🤷‍♂️